DQM/L1TMonitor/python/L1TStage2Emulator_cff.py

```diff
@@ -130,6 +130,7 @@
 valGtStage2Digis = simGtStage2Digis.clone(
     ExtInputTag = "gtStage2Digis",
     MuonInputTag = "gtStage2Digis:Muon",
+    MuonShowerInputTag = "gmtStage2Digis:MuonShower",
```

Comment:
Once https://github.com/cms-sw/cmssw/pull/40822 is confirmed working and merged I think one should use the gt digis here, right?